packages/dialtone-css/lib/build/less/components/emoji.less

```diff
@@ -1,4 +1,4 @@
 .d-emoji {
-  display: inline-block;
-  vertical-align: middle;
+  display: inline-flex;
+  flex: none;
 }
```

Comment:
hmm.. slightly worried this could break stuff?

Reply:
I think this change is not necessary, reverting this one, I just need to change this on emoji-text-wrapper in case the display property needs to change.